src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -125,6 +106,16 @@ protected override void RegisterOnAfterPackageLoadedAsyncWork(PackageLoadTasks a
 
         Task OnAfterPackageLoadedBackgroundThreadAsync(PackageLoadTasks afterPackageLoadedTasks, CancellationToken cancellationToken)
         {
+            var colorSchemeApplier = ComponentModel.GetService<ColorSchemeApplier>();
+            colorSchemeApplier.RegisterInitializationWork(afterPackageLoadedTasks);
+
+            // We are at the VS layer, so we know we must be able to get the IGlobalOperationNotificationService here.
+            var globalNotificationService = this.ComponentModel.GetService<IGlobalOperationNotificationService>();
+            Assumes.Present(globalNotificationService);
+
+            _solutionEventMonitor = new SolutionEventMonitor(globalNotificationService);
+            TrackBulkFileOperations(globalNotificationService);
```

Comment:
consider inlining this.  it's more confusing to me as a call out to a function.

Reply:
I think I'd prefer to keep that method separate. It's fairly long and has two local functions inside it already.